### ISSUE TITLE
Encoding problem when using Amazon Translate

### ISSUE THREAD

**Issue by alex23lemm** [paws-r/paws#287]
First of all thanks a lot for creating the `paws` package!

I encountered an encoding problem when using  `paws`' Amazon Translate client under Windows.  The issue is not present when using `boto3` via `reticulate`. 

Instead of an expected character, a sequence of Latin characters are shown. In the reprex below "ñ" is incorrectly encoded as "Ã±". Just make sure to modify the env name in the `use_condaenv()` call.

``` r
library(paws)

translate <- translate()
translated_text <- translate$translate_text(
  Text = "I go home tomorrow.",
  SourceLanguageCode = "en",
  TargetLanguageCode = "es"
)

translated_text$TranslatedText
#> [1] "Me voy a casa maÃ±ana."


```

``` r
library(reticulate)
 
use_condaenv("amazon-r", required = TRUE)
boto <- import("boto3")
translate <- boto$client("translate")

translated_text <- translate$translate_text(
  Text = "I go home tomorrow.",
  SourceLanguageCode = "en",
  TargetLanguageCode = "es"
)
 
translated_text$TranslatedText
#> [1] "Me voy a casa mañana."
```
According to [this page](https://www.i18nqa.com/debug/bug-utf-8-latin1.html) (which I found helpful) the underlying problem is that UTF-8 Bytes are treated as Windows-1252 or ISO-8859-1. When looking at the [UTF-8 Encoding Debugging Chart](https://www.i18nqa.com/debug/utf8-debug.html) you can find the entry for the encoding problem described above ("ñ" as  "Ã±").

Does `paws` modify the encoding before passing the API response?

**Comment by davidkretch** [comment]
Ah yeah, thank you for pointing this out.  Paws converts the raw bytes to character but fails to set the encoding.  It looks like this might be an easy fix.

**Comment by davidkretch** [comment]
Hello, I've added what I believe will fix this issue.  We'll also try to get it on CRAN in a couple weeks.

If you want to try it out, you can download the latest version of the request handling package `paws.common` with `remotes::install_github("paws-r/paws/paws.common")`.  If you do try it out, let us know if you have any issues with it.

Thank you!

**Comment by alex23lemm** [comment]
Hi David, thanks for the quick feedback and the fix. I can confirm that the encoding now works as expected when retrieving the API response. Thx!

**Comment by davidkretch** [comment]
Hello, sorry for the delay. This bug fix is now on CRAN. You'll want to update your `paws.common` to 0.3.2 to get the bug fix. Let me know if you have any issues.

Thank you!

**Comment by alex23lemm** [comment]
Thanks for note, David. I can confirm that the 0.3.2 version of  `paws.common` works as expected after having installed it from CRAN. 
I guess the only missing piece now is to add `paws.common` to the Imports list of `paws` in the next `paws` release where it is missing. Otherwise users will always need to install `paws.common` explicitly or get an error, for instance, when they try to create certain clients:

``` r
library(paws)
translate <- translate() 
# Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) :    there is no package called ‘paws.common’
```

**Comment by davidkretch** [comment]
Is that example code causing an error now? `paws` depends on a collection of packages including `paws.compute`, `paws.storage`, etc., and each of those imports `paws.common`, so installing `paws` should also install `paws.common`.

**Comment by alex23lemm** [comment]
Hi David, my bad. After the latest CRAN release of  `paws.common`, I just deleted and re-installed `paws` and `paws.common` to test if also the CRAN release works as expected. Since `paws` itself does not import `paws.common`, the test above on my end failed.
Your explanation above gave me the hint: Everything now works as expected on my end after I removed **all** paws-related packages (`paws` + `paws.[x]`) and re-installed `paws` from CRAN.
